docs/source/format/ADBC.rst

```diff
@@ -117,185 +50,7 @@ backwards-incompatible versions such as 2.0.0, but which still
 implement the API standard version 1.0.0.
 
 Similarly, this documentation describes the ADBC API standard version
-1.0.0.  If/when an ABI-compatible revision is made
+1.1.0.  If/when an ABI-compatible revision is made
 (e.g. new standard options are defined), the next version would be
-1.1.0.  If incompatible changes are made (e.g. new API functions), the
+1.2.0.  If incompatible changes are made (e.g. new API functions), the
 next version would be 2.0.0.
-
-Related work
-============
```

Comment:
I hate to see all this great "Related work" content being deleted. Should we move it to a "Related work" page on the ADBC site? Some of this is already covered in the ADBC FAQ, but not all of it.

Reply:
I'll move it!